backend/settings.py

```diff
@@ -210,3 +214,27 @@
         "toolbar": "formatselect | bold italic backcolor | "
             "bullist numlist outdent indent | removeformat | help",
     }
+
+# OAuth Settings, get these from the environment
+# TODO: Add some of these to the Django LTI model
```

Comment:
If you want to put these in the database, I'd probably lean towards making it a separate model that you can relate to the LTI tool config one if you want -- just to keep things flexible and not have to extend a model someone else created.

Reply:
Yeah, I'm not sure how we'll do this yet. This would be for a future phase anyway. I think I created an issue for it. #120